encord/objects/common.py

```diff
@@ -803,3 +803,48 @@ def _handle_wrong_number_of_found_items(
 AttributeClasses = Union[RadioAttribute, ChecklistAttribute, TextAttribute, Attribute]
 OptionTypes = Union[Type[FlatOption], Type[NestableOption], Type[Option]]
 OptionClasses = Union[FlatOption, NestableOption, Option]
+
+
+class DeidentifyRedactTextMode(Enum):
+    REDACT_ALL_TEXT = "REDACT_ALL_TEXT"
+    REDACT_NO_TEXT = "REDACT_NO_TEXT"
+    REDACT_SENSITIVE_TEXT = "REDACT_SENSITIVE_TEXT"
+
+
+class SaveDeidentifiedDicomConditionType(Enum):
+    NOT_SUBSTR = "NOT_SUBSTR"
+    IN = "IN"
+
+
+@dataclass
+class SaveDeidentifiedDicomConditionIn:
+    value: list[str]
```

Comment:
```suggestion
    value: List[str]
```
unfortunately we don't have an automatic check for these at the moment